contents/docs/product-analytics/funnels.mdx

```diff
@@ -37,9 +37,9 @@ For every flow throughout your product, more people will start it than complete
 
 **Step 1: Open up the funnels insight**
 
-Using the side menu - navigate to **Explore** and then **Funnels** - this will present you with a blank funnel screen.
+Using the side menu, navigate to **Insights** and then click the dropdown on the **New insight** button. This will present you with a blank funnel screen.
```

Comment:
```suggestion
Using the side menu, navigate to **Insights**, click the dropdown on the **New insight** button, and then select **Funnel**. This will present you with a blank funnel screen.
```